BTrees/Interfaces.py

```diff
@@ -277,6 +277,10 @@ class IMerge(Interface):
     :meth:`BTrees.IIBTree.IIBTree.union` can only be used with :class:`~BTrees.IIBTree.IIBTree`,
     :class:`~BTrees.IIBTree.IIBucket`, :class:`~BTrees.IIBTree.IISet`, and :class:`~BTrees.IIBTree.IITreeSet`.
 
+    The number protocols methods ``__and__``, ``__or__`` and ``__sub__`` are provided
+    by all the data structures. They are shortcuts for :meth:`~BTrees.Interfaces.IMerge.intersection`,
+    :meth:`~BTrees.Interfaces.IMerge.union` and :meth:`~BTrees.Interfaces.IMerge.difference`.
+
```

Comment:
I just set this line of documentation because I was not sure on which interface to add the new operators. Is it ok like this? Or should I add the operators to the IBTree, ITreeSet and ISet interfaces?

Reply:
It's the modules (e.g., `BTrees.OOBTree`) that implement `IMerge`, so I don't think the method definitions of the operators belong here. It doesn't hurt anything to mention in the main docstring here that there are operator synonyms for the functions defined by this interface, but I think that the operator method definitions should be moved to the appropriate datatype interfaces.

Reply:
So, just to be sure, IBTree, ITreeSet and ISet ?

Reply:
That sounds right to me. (I don't think there's an `IBucket`.)